src/credentials/FabricTable.cpp

```diff
@@ -683,10 +714,11 @@ CHIP_ERROR FabricTable::AddNewFabricInner(FabricInfo & newFabric, FabricIndex *
     {
         if (!fabric.IsInitialized())
         {
+            NotBeforeCollector notBeforeCollector;
```

Comment:
We're going to need a followup to do this with UpdateNOC too, after https://github.com/project-chip/connectedhomeip/pull/18866 merges.

Reply:
#18866 is merged

Reply:
Done.